models.py

```diff
@@ -55,6 +56,38 @@ def format_description(field, value):
         name = localization.get_field(field).decode(db_encoding)
         return u"{0}: {1}".format(name, value)
 
+class HighlightPoint(Point, Base):
+    __tablename__ = "highlight_markers"
+    __table_args__ = (
+        Index('highlight_long_lat_idx', 'latitude', 'longitude'),
```

Comment:
I was thinking that in the future, when we decide to make some scene from all the locations, to use an index table so particular querying will be faster, actually I'm not sure how we want to query the highlights table in the future, **so should I remove it?**
P.S I'm working on creating a R*Tree module, for better location querying, for the accidents bounding queries, It should be usable here to.